app/utils/avoid_native_bridge.js

```diff
@@ -0,0 +1,10 @@
+
+import {Platform} from 'react-native';
+
+export default async function avoidNativeBridge(runOptimized, optimized, fallback) {
```

Comment:
Could you include a comment explaining what this does? It's not clear what it does from the name

Also, does this need to be an async function? Could we just make it a regular function without awaiting the fallback function since it will return the same promise anyway?